### ISSUE TITLE
Fix/improve geojson output of WRT

### ISSUE THREAD

**Issue by MartinPontius** [52North/MariGeoRoute#18]
1. Use `properties` instead of `property` ([WRT source code](https://github.com/52North/MariGeoRoute/blob/main/WeatherRoutingTool/WeatherRoutingTool/routeparams.py#L108))
2. Add feature id [(ref),](https://github.com/qgis/QGIS/issues/40805) i.e
 ```
     {
      "type": "Feature",
      "id": "my_id",
      "geometry": {
        "type": "Point",
        "coordinates": [
          -9753562.172631999477744,
          5141014.504863454028964
        ]
      }
    },
```
3. Add ship id?

**Comment by bhubneshmaharana** [comment]
Hey MartinPontius,
is anyone working on this issue or should I open a PR on this .

**Comment by MartinPontius** [comment]
Issue is solved in commits [531f99d988d23c0b1e82ef24e05c210a37cb754e](https://github.com/52North/MariGeoRoute/commit/531f99d988d23c0b1e82ef24e05c210a37cb754e) and [4ae8a2c3ae292cd9d3e2d4da7f346f5f406b1cc9](https://github.com/52North/MariGeoRoute/commit/4ae8a2c3ae292cd9d3e2d4da7f346f5f406b1cc9)
